Fix aclocal warnings:
/usr/share/aclocal/pth.m4:43: warning: underquoted definition of _AC_PTH_ERROR
/usr/share/aclocal/pth.m4:54: warning: underquoted definition of _AC_PTH_VERBOSE
/usr/share/aclocal/pth.m4:60: warning: underquoted definition of AC_CHECK_PTH
--- pth-2.0.3/pth.m4
+++ pth-2.0.3/pth.m4
@@ -42,3 +42,3 @@
 dnl #   auxilliary macros
-AC_DEFUN(_AC_PTH_ERROR, [dnl
+AC_DEFUN([_AC_PTH_ERROR], [dnl
 AC_MSG_RESULT([*FAILED*])
@@ -53,3 +53,3 @@
 ])
-AC_DEFUN(_AC_PTH_VERBOSE, [dnl
+AC_DEFUN([_AC_PTH_VERBOSE], [dnl
 if test ".$verbose" = .yes; then
@@ -59,3 +59,3 @@
 dnl #   the user macro
-AC_DEFUN(AC_CHECK_PTH, [dnl
+AC_DEFUN([AC_CHECK_PTH], [dnl
 dnl
